rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -152,5 +152,14 @@
               </plugins>
           </build>
       </profile>
+      <profile>
+        <id>quarkus-uber</id>
+        <activation>
+          <activeByDefault>true</activeByDefault>
+        </activation>
+        <properties>
+          <quarkus.package.type>uber-jar</quarkus.package.type>
```

Comment:
just curious, did you try if it works with fast-jar?

Reply:
I'll give it a go

Reply:
It's nice but I've decided to go with the standard maven packaging for this one. If someone wants to raise a PR for it then please do but bear in mind that we are pulling in the resulting uber jar from the [quickstart](https://github.com/mmusgrov/quickstart-1/blob/JBTM-3430/rts/lra-examples/lra-coordinator/pom.xml#L63). It wasn't clear how to package up the resulting jar (target/quarkus-app/quarkus-run.jar) into the lra-coordinator-quarkus artefact for consumption by the quickstart.

Reply:
sure, we can take this later